integration_tests/integration/barcode/selectEnvelopeSize.spec.ts

```diff
@@ -15,6 +15,8 @@ context('Select envelope size', () => {
   })
 
   it('should goto print coversheets given envelope size selected', () => {
+    cy.task('stubCreateBarcode')
```

Comment:
The barcode number is now generated slightly earlier in our process, so need this stub in this test